natives.json

```diff
@@ -89734,35 +89734,35 @@
 			"gta_jhash": "0xFFFC85D4"
 		},
 		"0x274EE1B90CFA669E": {
-			"name": "_0x274EE1B90CFA669E",
+			"name": "_IS_MODEL_A_OBJECT",
```

Comment:
1) Grammatically incorrect (it's _an_ object).
2) While this does check for some kind of object type it doesn't exactly check if the model is an object. Please do some more research on this or leave it unnamed for now.